google/cloud/internal/tracing_rest_client_test.cc

```diff
@@ -191,9 +193,9 @@ TEST(TracingRestClient, HasCloudTraceContext) {
   EXPECT_THAT(contents, IsOkAndHolds(MockContents()));
 
   auto spans = span_catcher->GetSpans();
-  EXPECT_THAT(spans,
-              UnorderedElementsAre(SpanNamed("HTTP/PATCH"), SpanNamed("Read"),
-                                   SpanNamed("Read")));
+  EXPECT_THAT(
+      spans, UnorderedElementsAre(SpanNamed("HTTP/PATCH"), SpanNamed("Read"),
+                                  SpanNamed("SendRequest"), SpanNamed("Read")));
```

Comment:
nit: I know the expectation is unordered, but I would put the `SendRequest` before the `Read`s.

Reply:
Fixed